ansible/roles/ooni-explorer-next/templates/ngx-explorer

```diff
@@ -26,6 +26,10 @@ server {
     access_log  /var/log/nginx/{{ explorer_domain }}.access.log; # FIXME: log_format
     error_log   /var/log/nginx/{{ explorer_domain }}.error.log;
 
+    location /data/ {
+        root {{ explorer_country_data_path }};
+    }
+
```

Comment:
FYI.
I understand the convenience of that during development (and that's, unfortunately, the only way to do that right now). But also, I would advice against doing alike stuff on the production explorer setup.
I'd suggest permanent caching of specific URL prefix instead (with versioned URLs), or non-permanent caching and cache destruction on roll-out.
The reason is: it's way easier to build, roll-out and roll-back the container as a whole, than to have some bits of stateless web-app in the container and some bits on the filesystem.
Please, document that it's dev-only crutch in the config file If you agree with that, so it does not end up in the production setup "by accident". If you disagree, let's discuss :)

Reply:
Also, beware of [off-by-slash](https://twitter.com/x0rz/status/1052899891624710145). :)

Reply:
Yes I agree this is a dev only "crutch". I think adding a simple comment in the ngx-explorer config file saying "# XXX this is for development only".

Reply:
Wow. I had to decide whether the slash is necessary or not and ended up dropping it after looking at all the examples from NGINX documentation. None of them use a `/` for directory paths.

Added the comment too.

Reply:
@sarathms I'm unsure if it applies to `root`, or just to `alias`. I don't have in-deph understanding of that vulnerability.